build/Makefile

```diff
@@ -480,7 +484,8 @@ STATIC_PROVIDER_UNITFILES = \
 	$(PROVIDER_TEST_DIR)/sqlstoredproc_test.cpp \
 	$(PROVIDER_TEST_DIR)/support/testrunnerlogpolicy.cpp \
 	$(TEST_DIR)/myauth/myauth_test.cpp \
-	$(PROVIDER_TEST_DIR)/providertestutils.cpp
+	$(PROVIDER_TEST_DIR)/providertestutils.cpp \
+	$(PROVIDER_TEST_DIR)/config_scanner_test.cpp
```

Comment:
Won't matter for build, but this should be moved higher up in the list of tests. The intent was that tests themselves are first, and support files follow. Given that, providertestutils.cpp should probably be the last one. And that said, myauth_test.cpp (which you didn't touch) should be moved up too, since testrunnerlogpolicy.cpp is a support file. Very minor nit.